bin/yarn

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
```

Comment:
I think `make image` should always be step 1 for local dev, so that's a good thing to call out in doc. I don't think a bin stub like this should make the image itself (we don't do that in other similar bin stubs).

The thing about channels & local images & such is a shame, but that's messy *today* as well, so I think we can leave that yak hairy for the time being.